src/main/java/001-Introduction.adoc

```diff
@@ -58,13 +62,13 @@ Prerequisites for using the {product-name} API:
 * A client or programming library that initiates and receives HTTP requests
 from the API server. For example:
 
-** The https://github.com/oVirt/ovirt-engine-sdk/tree/master/sdk[oVirt Python SDK].
+** The link:https://github.com/oVirt/ovirt-engine-sdk/tree/master/sdk[oVirt Python SDK].
```

Comment:
Python-sdk project has recently been split, this link references what is now the 'generator' part; i.e, the logic for generating the python-sdk from the model. The actual SDK (+ usage examples) is now here: https://github.com/oVirt/python-ovirt-engine-sdk4

Reply:
Fixed.